src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java

```diff
@@ -1,50 +1,85 @@
 
-	@Test(description = "JIRA#DEMO-0011")
-	@MethodOwner(owner = "qpsdemo")
-	public void testLoginUser() {
-		setApplicationPath();
-		String username = "Test user";
-		String password = RandomStringUtils.randomAlphabetic(10);
-		WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
-		Assert.assertTrue(welcomePage.isPageOpened(), "Welcome page isn't opened");
-		LoginPageBase loginPage = welcomePage.clickNextBtn();
-		Assert.assertFalse(loginPage.isLoginBtnActive(), "Login button is active");
-		loginPage.typeName(username);
-		loginPage.typePassword(password);
-		loginPage.selectMaleSex();
-		loginPage.checkPrivacyPolicyCheckbox();
-		CarinaDescriptionPageBase carinaDescriptionPage = loginPage.clickLoginBtn();
-		Assert.assertTrue(carinaDescriptionPage.isPageOpened(), "Carina description page isn't opened");
-	}
-	
-	private void setApplicationPath() {
-		String pathToApp;
-		String iosPath = "src/main/resources/carinademoexample.zip";
-		String androidPath = "src/main/resources/carinademoexample.apk";
-		if (Type.ANDROID_PHONE.getFamily().equalsIgnoreCase(R.CONFIG.get("capabilities.platformName")))
-			pathToApp = androidPath;
-		else 
-			pathToApp = iosPath;
-		File file = new File(pathToApp);
-		Configuration.setMobileApp(file.getAbsolutePath());
-	}
+    @BeforeClass
+    public void setup() {
+        String pathToApp = R.CONFIG.get("application.path");
+        Configuration.setMobileApp(new File(pathToApp).getAbsolutePath());
+    }
+
+    @Test(description = "JIRA#DEMO-0011")
+    @MethodOwner(owner = "qpsdemo")
+    public void testLoginUser() {
+        String username = "Test user";
+        String password = RandomStringUtils.randomAlphabetic(10);
+        WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
+        Assert.assertTrue(welcomePage.isPageOpened(), "Welcome page isn't opened");
+        LoginPageBase loginPage = welcomePage.clickNextBtn();
+        Assert.assertFalse(loginPage.isLoginBtnActive(), "Login button is active when it should be disabled");
+        loginPage.typeName(username);
+        loginPage.typePassword(password);
+        loginPage.selectMaleSex();
+        loginPage.checkPrivacyPolicyCheckbox();
+        CarinaDescriptionPageBase carinaDescriptionPage = loginPage.clickLoginBtn();
+        Assert.assertTrue(carinaDescriptionPage.isPageOpened(), "Carina description page isn't opened");
+    }
+
+    @Test(description = "JIRA#DEMO-0011")
+    @MethodOwner(owner = "qpsdemo")
+    public void testWebView() {
+        WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
+        LoginPageBase loginPage = welcomePage.clickNextBtn();
+        loginPage.login();
+        WebViewPageBase webViewPageBase = initPage(getDriver(), WebViewPageBase.class);
+        MobileContextUtils contextHelper = new MobileContextUtils();
+        contextHelper.switchMobileContext(MobileContextUtils.View.WEB, getDriver());
```

Comment:
contextHelper.switchMobileContext(MobileContextUtils.View.WEB, getDriver()); it looks strange.
I believe it should be like that:
contextHelper.switchMobileContext(View.WEB);